modules/nml.nf

```diff
@@ -18,3 +18,30 @@ process generateIridaReport {
     irida_samples.py --sample_info ${samplecsv} --prefix ${params.prefix} --sample_dir irida_upload
     """
 }
+
+process runNcovTools {
+
+    publishDir "${params.outdir}/irida_upload", pattern: "*.pdf", mode: "copy"
+
+    //conda 'environments/ncovtools.yml'
+
+    input:
+    file(config)
+    file(reference)
+    file(amplicon)
+    file(nanopolishresults)
+
+    output:
+    file("*.pdf")
+
+    script:
+    """
+    git clone https://github.com/jts/ncov-tools.git
+    mv ${config} ${reference} ${amplicon} ./ncov-tools
+    mkdir ./ncov-tools/run
+    mv ${params.prefix}* ./ncov-tools/run
+    cd ncov-tools
+    snakemake -s qc/Snakefile all_qc_sequencing --cores 6
```

Comment:
Can you get value for --cores # from https://github.com/phac-nml/ncov2019-artic-nf/blob/master/conf/coguk/nml.config#L6

Perhaps a new label so we can adjust as needed without changing nextflow code.

Reply:
Do you want me to look into a way to grab the value from the config file or just create a new label and match the cores to that?

Reply:
Believe I addressed this in the way you wanted